packages/shared/src/utils/date/date-time.js

```diff
@@ -229,3 +229,32 @@ export const formatMiliseconds = (miliseconds, str_format) => moment.utc(milisec
  */
 export const convertDateFormat = (date, from_date_format, to_date_format) =>
     moment(date, from_date_format).format(to_date_format);
+
+/**
+ * returns duration in seconds
+ * @param {String} duration formatted duration string
+ */
+export const durationToSecond = duration => {
```

Comment:
what's the format of duration here exactly? 🤔

Reply:
Its something like 46s, 23d etc. But since the method which is using `durationToSecond()` is not used elsewhere, I have got rid of it from shared utils in the latest commit. 😄